File: src/main/java/com/bdev/hogwarts_api/dao/StudentPaymentDao.java
package com.bdev.hogwarts_api.dao;

import com.bdev.hogwarts_api.data.model.student_payment.StudentPaymentModel;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface StudentPaymentDao extends JpaRepository<StudentPaymentModel, Long> {
    List<StudentPaymentModel> getAllByStudentId(long studentId);
}

File: src/main/java/com/bdev/hogwarts_api/dao/StudentPaymentDao.kt
package com.bdev.hogwarts_api.dao

import com.bdev.hogwarts_api.data.model.student_payment.StudentPaymentModel
import org.springframework.data.jpa.repository.JpaRepository

interface StudentPaymentDao : JpaRepository<StudentPaymentModel, Long> {
    fun getAllByStudentLogin(studentLogin: String): List<StudentPaymentModel>
}
